conf/base_config.yaml

```diff
@@ -129,7 +131,6 @@ training:
         check_val_every_n_epoch: 10
         log_every_n_steps: 10  # corresponds to n_batches
         max_epochs: ${training.n_epochs}
-        min_epochs: ${training.n_epochs}
```

Comment:
Is there a way to keep `min_epochs` while having the option to enable early stopping? Ideally we would want to still be able to train for exactly >= `min_epochs`?

Reply:
We can keep it, but we'd have to make it a separate config parameter - with it just using training.n_epochs, it would always train for exactly n_epochs, even with early stopping enabled.